shuttleofx_client/Gruntfile.js

```diff
@@ -15,7 +15,6 @@ module.exports = function (grunt) {
     pkg: grunt.file.readJSON('package.json'),
     project: {
       static: 'static_tmp',
-      static_dist: 'static_dist',
       static_tmp: 'static'
```

Comment:
If you remove it, it's better to have only static and statis_dist I think.
But it can be necessary to rename the folder in the source projet.